src/wallets/metamask/actions/unlock.ts

```diff
@@ -1,13 +1,13 @@
 import { Page } from 'playwright-core';
-import { clickOnButton, typeOnInputField } from '../../../helpers';
 import { closePopup } from '../setup/setupActions';
 
 export const unlock =
   (page: Page) =>
   async (password = 'password1234'): Promise<void> => {
     await page.bringToFront();
 
-    await typeOnInputField(page, 'Password', password);
-    await clickOnButton(page, 'Unlock');
+    await page.getByTestId('unlock-password').fill(password);
+    await page.getByTestId('unlock-submit').click();
```

Comment:
Are the testIds something added in the newer metamask? Wondering why the switch away from using the helpers.

Reply:
Yes, they are starting to add them so moving away from the xpaths will make the code more digestible.